template/src/test/java/com/asyncapi/ApplicationTests.java

```diff
@@ -1,16 +0,0 @@
-package {{ params['userJavaPackage'] }};
```

Comment:
Why removed?

Reply:
If we no longer need the @SpringBootApplication class, we also dont need its test class...